go/vt/vttablet/onlineddl/vrepl.go

```diff
@@ -60,13 +62,12 @@ type VReplStream struct {
 }
 
 // livenessTimeIndicator returns a time indicator for last known healthy state.
-// vreplication uses two indicators: time_updates and time_heartbeat. Either one making progress is good news. The greater of the two indicates the
-// latest progress. Note that both indicate timestamp of events in the binary log stream, rather than time "now".
-// A vreplication stream health is determined by "is there any progress in either of the two counters in the past X minutes"
+// vreplication uses three indicators:
+// - transaction_timestamp
+// - time_heartbeat
+// - time_throttled.
+// Update of any of them, also updates time_updated, indicating liveness.
```

Comment:
`Updating` any of them, right?

Reply:
I didn't know that, but you're the native English speaker :sweat_smile: